src/fixtures/openapi/types.ts

```diff
@@ -0,0 +1,231 @@
+export interface Prop {
```

Comment:
This is where that generic API client would be great to have :100:

Reply:
Exactly!